generate_graph.py

```diff
@@ -87,16 +87,21 @@ def plot_results(columns, df):
 
 if __name__ == '__main__':
     # Load Raw Density Data
-    data_dir = '/data6/lekevin/hab-spc'
+    data_dir = '/data6/SnehaKondur/hab-spc'
     fname = os.path.join(data_dir, 'rawdata/Density_data.csv')
     if not os.path.exists(fname):
         print('Creating density data')
-        create_density_csv()
+        create_density_csv(fname)
     df = pd.read_csv(fname)
     m, n = df.shape
 
     # Clean up nan values
     df = df.dropna()
+
+    # Setting a threshold value to filter the microscopy data.
+    threshold = 40000
+    # Filtering the df based on micro_total_phyto values
+    df = df[df.micro_proro < 40000]
```

Comment:
You validated that 5 rows are dropped? How does the graph look after removing the outliers?